workspaces/workspaces-as-code/templates.md

```diff
@@ -64,13 +64,23 @@ workspace:
           command: |
             apt update
             apt install -y curl
+        # Be careful keyscans like this!
```

Comment:
is this comment misplaced? seems like it belongs above the "Add github to known hosts" step?

Reply:
Whoops yes. That's what I get for copy pasting from my larger template example